api/v3/examples/Activity/ContactRefCustomField.php

```diff
@@ -10,7 +10,7 @@
 function activity_create_example() {
   $params = array(
     'source_contact_id' => 1,
-    'activity_type_id' => '51',
+    'activity_type_id' => '55',
```

Comment:
@eileenmcnaughton is there any activity type id with 55? I couldn't find any in xml/templates/civicrm_data.tpl.

I raised this since a test is failing for one of my PR https://test.civicrm.org/job/CiviCRM-Core-PR/13838/testReport/junit/(root)/api_v3_ActivityTest/testActivityCreateExample/